clang/docs/ReleaseNotes.rst

```diff
@@ -49,6 +49,9 @@ C++ Specific Potentially Breaking Changes
   few users and can be written as ``__is_same(__remove_cv(T), decltype(nullptr))``,
   which GCC supports as well.
 
+- The warning `-Wenum-constexpr-conversion` has been upgraded into a hard
+  compiler error that cannot be suppressed, as required by the C++ Standard.
```

Comment:
Because this is a breaking change, it might be nice to have a code example showing what code is impacted (like we did in https://github.com/llvm/llvm-project/blob/release/16.x/clang/docs/ReleaseNotes.rst?plain=1#L170).

Reply:
That's a great formulation so I'll go straight ahead and copy it here, thanks! Added further clarification